AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -45,6 +45,13 @@ @implementation MSAnalytics
 
 #pragma mark - Service initialization
 
++ (void)load {
+  [MSAppCenterUserDefaults addKeysToMigrate:@{
```

Comment:
There's one tricky set of keys to migrate here `AnalyticsIsEnabledKey/xxxx`, the transmission targets enabled state keys. It's dynamically created with the target key from the token. I we'll need to list all of them from the user defaults but not sure about the state of the defaults at load time.